wdl/ExpansionHunter.wdl

```diff
@@ -251,6 +297,21 @@ task ConcatEHOutputs {
 
         merge_tsv "~{write_lines(alleles_tsvs)}" "~{output_prefix}_alleles.tsv"
         merge_tsv "~{write_lines(variants_tsvs)}" "~{output_prefix}_variants.tsv"
+
+        gzip "~{output_prefix}_alleles.tsv"
+        gzip "~{output_prefix}_variants.tsv"
+
+        python /opt/str/merge_csv_files.py \
+            --input-filename ~{write_lines(sample_metrics_)} \
+            --metrics metrics.csv \
+            --missing-metrics missing_metrics.csv
+
+        # Combine multiple archives into one archive,
+        # by first extracting the files then re-archiving it.
+        # This allows preserving a flat structure with one archive,
+        # otherwise, it would create an archive containing archives.
+        mkdir temp_dir
+        cat ~{write_lines(reviewer_images_gz_)} | xargs -I{} tar -xzvf {} -C temp_dir/ && tar -czvf reviewer_images.tar.gz -C temp_dir/ .
```

Comment:
I may be mistaken here, but I think you need to delete the extracted files in `temp_dir/` for each line in `reviewer_images_gz_`? It seems like the temp dir will accumulate files and keep re-tarring all previous files? You may need to use a loop here.

Reply:
Actually in my experience, it's been fastest to just extract everything into one temp dir, then do one big `tar -c` to create a new archive rather than adding files incrementally.

Reply:
Good suggestion, thanks! I updated it accordingly.